src/tmpfiles/tmpfiles.c

```diff
@@ -2260,6 +2283,69 @@ static int remove_item(Item *i) {
         }
 }
 
+static int format_age_by(const Item *item, char **ret_age_by_fmt) {
+        _cleanup_free_ char *fmt = NULL;
+        _cleanup_strv_free_ char **split = NULL;
+        static const struct {
+                AgeBy age_by_flag;
+                const char *age_by_str;
+        } age_by_types[] = {
+                { AGE_BY_ATIME, "atime" },
+                { AGE_BY_BTIME, "btime" },
+                { AGE_BY_CTIME, "ctime" },
+                { AGE_BY_MTIME, "mtime" },
+        };
+
+        assert(ret_age_by_fmt);
+        assert(item);
+
+        if (item->age_by_file == AGE_BY_DEFAULT_FILE &&
+            item->age_by_dir == AGE_BY_DEFAULT_DIR) {
+                char *tmp = NULL;
+
+                tmp = strdup("");
+                if (!tmp)
+                        return -ENOMEM;
+
+                *ret_age_by_fmt = tmp;
+                return 0;
+        }
+
+        for (size_t i = 0; i < ELEMENTSOF(age_by_types); i++) {
+                char *tmp = NULL;
+
+                if FLAGS_SET(item->age_by_file, age_by_types[i].age_by_flag) {
```

Comment:
there's a `()` missing here... 

i mean, it guess it makes sense this compiles without it, but no need to be this confusing needlessly

Reply:
Oops, definitely missed that. Thanks, will update!

Reply:
This function has been replaced with `age_by_from_string`.